AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1117,23 +1115,7 @@ - (ASScrollDirection)_scrollDirectionForVelocity:(CGPoint)scrollVelocity
 
 - (ASScrollDirection)scrollableDirections
 {
-  //Cache results of layoutInspector to ensure flags are up to date if getter lazily loads a new one.
-  id<ASCollectionViewLayoutInspecting> layoutInspector = self.layoutInspector;
-  if (_layoutInspectorFlags.scrollableDirections) {
-    return [layoutInspector scrollableDirections];
-  } else {
```

Comment:
I'm of the opinion the code was dangerous. It's correct most of the time, but not when the bound size has changed and relayout to update contentSize hasn't occurred yet (this is what started us on this path, we were checking scrollable directions to see if we should invalidate all the layout).